student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] Its a requirement that the constructor extends parent's constructor by adding @classroom and a parameter for it.
e.g
![image](https://user-images.githubusercontent.com/81932478/192602273-311e2e5a-abf9-4487-9e98-1b9d6981d6fc.png)